tests/test_hydra.py

```diff
@@ -440,16 +444,77 @@ def test_cfg(tmpdir: Path, flag: str, expected_keys: List[str]) -> None:
         param(["--cfg=job", "--package=db.driver"], "mysql\n", id="package=db.driver"),
     ],
 )
-def test_cfg_with_package(tmpdir: Path, flags: List[str], expected: str) -> None:
+def test_cfg_with_package(
+    tmpdir: Path, flags: List[str], resolve: bool, expected: str
+) -> None:
     cmd = [
         "examples/tutorials/basic/your_first_hydra_app/5_defaults/my_app.py",
         "hydra.run.dir=" + str(tmpdir),
     ] + flags
+    if resolve:
+        cmd.append("--resolve")
 
     result, _err = run_python_script(cmd)
     assert normalize_newlines(result) == expected.rstrip()
 
 
+@mark.parametrize(
+    "resolve,expected",
+    [
+        param(
+            False,
+            dedent(
+                """\
+                node:
+                  loompa: 10
+                  zippity: ${node.loompa}
+                  do: oompa ${node.loompa}
+                  waldo: ???
+                """
+            ),
+            id="cfg",
```

Comment:
a bit too big.
I think something as  simple as 
```python
{"a": "${b}", "b": 10}
```

will be enough to test both behaviors.

Reply:
Ok, I've made the change in https://github.com/facebookresearch/hydra/pull/1616/commits/4d1cbd79f454f4e40cee851a29f56d8f1dd96952.